doc/dev_guide.rst

```diff
@@ -132,34 +148,49 @@ committers. Unlike :ref:`sec:code-style-guidelines`, this section can be
 entirely ignored.
 
 - Keep function length to a minimum.
-	As mentioned `at this talk <http://www.infoq.com/presentations/Scrub-Spin>`_, `MSL <http://en.wikipedia.org/wiki/Mars_Science_Laboratory>`_ included the rule that no function should be longer than 75 lines of code.
+	As mentioned `at this talk <http://www.infoq.com/presentations/Scrub-Spin>`_,
+    `MSL <http://en.wikipedia.org/wiki/Mars_Science_Laboratory>`_
```

Comment:
folded lines should have consistent indentation. Otherwise, the reST parser as described in <http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#indentation>.

On line 152, the MSL link is 4 spaces to the left of the preceding line. When I build dev_guide.rst, it is rendered as a separate paragraph downward and to the left of the text "As mentioned at this talk,"

This comment applies to other cases of folded lines through line 194 of dev_guide.rst.